footer.php

```diff
@@ -23,8 +23,12 @@
 		</a>
 		<span class="sep"> | </span>
 		<?php
-			/* translators: 1: Theme name, 2: Theme author. */
-			printf( esc_html__( 'Theme: %1$s by %2$s.', 'wp-rig' ), '<a href="' . esc_url( 'https://github.com/wprig/wprig/' ) . '">WP Rig</a>', 'the contributors' );
+		/* translators: 1: Theme name, 2: Theme author. */
+		printf( esc_html__( 'Theme: %1$s by %2$s.', 'wp-rig' ), '<a href="' . esc_url( 'https://github.com/wprig/wprig/' ) . '">WP Rig</a>', 'the contributors' );
+
+		if ( function_exists( 'the_privacy_policy_link' ) ) :
+			the_privacy_policy_link( '<span class="sep"> | </span>', '' );
+		endif;
```

Comment:
Please use regular `if { }` syntax instead. The `if : endif;` is no longer preferred (also in WordPress core).

Reply:
Thank you. That's changed.